### PR TITLE
Added ListCommand

### PR DESCRIPTION
**Pull request by jasmoon**

### CODE REVIEW

**Comment by jasmoon** [comment]
#75 #76 #77

File: src/test/java/seedu/address/logic/commands/HelpCommandTest.java [hunk 1]
```diff
@@ -11,14 +12,40 @@
...
     }
+
+    @Test
+    public void execute_helpForTask_success() {
+        HelpCommand command = new HelpCommand("task");
+        assertCommandSuccess(command, TaskCommand.MESSAGE_USAGE);
+    }
+
+    @Test
+    public void execute_helpForDelete_success()    {
```
Comment (Kyomian) [inline src/test/java/seedu/address/logic/commands/HelpCommandTest.java]: Change this to execute_helpForRemove_success()

In fact, change all Delete to Remove

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -92,6 +91,35 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
         }
     }
 
+    //@@author jasmoon
+    /**
+     * Executes the given {@code command}, confirms that <br>
+     * - the result message matches {@code expectedMessage} <br>
+     */
+    public static void assertCommandSuccess(Command command, String expectedMessage)   {
```
Comment (Kyomian) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: Quite sly of you to include these two overloaded methods

File: src/test/java/seedu/address/logic/commands/HelpCommandTest.java [hunk 3]
```diff
@@ -11,14 +12,40 @@
...
+        HelpCommand command = new HelpCommand("delete");
+        assertCommandSuccess(command, DeleteCommand.MESSAGE_USAGE);;
+    }
+
+    @Test
+    public void execute_helpForComplete_success()    {
+        HelpCommand command = new HelpCommand("complete");
+        assertCommandSuccess(command, CompleteCommand.MESSAGE_USAGE);;
+    }
+
```
Comment (Kyomian) [inline src/test/java/seedu/address/logic/commands/HelpCommandTest.java]: I also included execute_helpForHelp()_success and execute_helpForMan()_success tests in my side of the code.
I commented them out for now, so after I push, you can uncomment them. 
Give me some credit :D

**Pull request merged**